### PR TITLE
TASK: Require correct version of flow-development-collection

### PR DESCRIPTION
**Pull request by johannessteu**

### CODE REVIEW

**Comment by kdambekalns** [comment]
Thanks, but be aware that this is generated from the individual manifests in the collection using a nice script @kitsunet came up with IIRC. So there should be two files being touched here…

**Comment by kdambekalns** [comment]
Ok, you need to adjust `.composer.json` to fix this in a durable way. :)

**Comment by aertmann** [comment]
@johannessteu you still need to adjust `.composer.json` in the `TYPO3.Neos` folder as @kdambekalns mentioned

**Comment by johannessteu** [comment]
@aertmann pushed fix

File: .composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "license": "GPL-3.0+",
   "type": "neos-package-collection",
   "require": {
-    "neos/flow-development-collection": "dev-master"
+    "neos/flow-development-collection": "~3.3.0"
```
Comment (aertmann) [inline .composer.json]: should be `3.3.x-dev`

**Comment by aertmann** [comment]
@johannessteu left a comment, also the PR needs to be rebased..

**Comment by aertmann** [comment]
@kdambekalns guess you can remove your 👎 now

**Comment by johannessteu** [comment]
Rebased, force pushed. Looks like the `.composer.json` and the root `composer.json` are already fixed in the meantime

**Pull request merged**